neurokit2/ecg/ecg_delineate.py

```diff
@@ -49,8 +49,8 @@ def ecg_delineate(
     sampling_rate : int
         The sampling frequency of ``ecg_signal`` (in Hz, i.e., samples/second). Defaults to 1000.
     method : str
-        Can be one of ``"peak"`` for a peak-based method, ``"cwt"`` for continuous wavelet transform
-        or ``"dwt"`` (default) for discrete wavelet transform.
+        Can be one of ``"peak"`` for a peak-based method, ``"prominence"`` for a peak-prominence-based method (Emrich et al., 2024),
```

Comment:
I think CI complains about the line length here

```suggestion
        Can be one of ``"peak"`` for a peak-based method, ``"prominence"`` for a peak-prominence-based 
        method (Emrich et al., 2024), ...
```